rest/config.go

```diff
@@ -1998,9 +2002,9 @@ func (sc *ServerContext) FetchConfigs(ctx context.Context, isInitialStartup bool
 }
 
 // _applyConfigs takes a map of dbName->DatabaseConfig and loads them into the ServerContext where necessary.
-func (sc *ServerContext) _applyConfigs(ctx context.Context, dbNameConfigs map[string]DatabaseConfig, isInitialStartup bool) (count int) {
+func (sc *ServerContext) _applyConfigs(ctx context.Context, dbNameConfigs map[string]DatabaseConfig, isInitialStartup bool, loadFromBucket bool) (count int) {
```

Comment:
I'm pretty sure `isInitialStartup` is the same flag logically as the new `loadFromBucket`. Is there anywhere where these two bools aren't the same?

Reply:
Its not the same for the goroutine that polls for new configs https://github.com/couchbase/sync_gateway/blob/65c5ae456a553e3018a3ae7ee938eaedde2f0174/rest/server_context.go#L2116 

My thinking was handling incorrect data migration scenarios where you copy dbconfig from on prem with different collections that what are defined in capella, then you would still fall into this handling? I could be going down the wrong rabbit hole there though and would accept if we should only fall into this handling on node startup.

Reply:
Oh I see - this also applies to nodes that load a newly created db config that was created on another node.